### PR TITLE
Feat : Explore page

### PR DESCRIPTION
**Pull request by aiman-ansari**
- Added filter by category functionality.
Screenshot (18)
Screenshot (19)

### CODE REVIEW

File: src/Components/Header/Header.css [hunk 1]
```diff
@@ -21,18 +21,7 @@ nav{
 nav.active{
     background-color: rgb(22, 22, 22);
 }
-.image{
-    height: 70vh;
-    width: 100%;
-    background: black;
-    overflow: hidden;    
-}
```
Comment (theVivek-kumar) [inline src/Components/Header/Header.css]: please gave  img tag a class

File: src/Components/Header/Header.css [hunk 2]
```diff
@@ -21,18 +21,7 @@ nav{
...
-img {
-    height: 70vh;
-    width: 100%;
-    object-fit: fill;
-    opacity: 0.3;
-}
+
 nav ul {
```
Comment (theVivek-kumar) [inline src/Components/Header/Header.css:35]: gave ul tag a class then style

File: src/Pages/Explore/Explore.css [hunk 3]
```diff
@@ -0,0 +1,36 @@
+.category-container{
```
Comment (theVivek-kumar) [inline src/Pages/Explore/Explore.css:1]: good clean code

File: src/Pages/Explore/Explore.js [hunk 4]
```diff
@@ -1,7 +1,69 @@
...
+        dispatch({
+            type:"ALL"
+        })
+    }
+   const sorted = Filter(videos, state)
+  
     return(
-        <div>
-            explore page
+        <div className="m-2">
```
Comment (theVivek-kumar) [inline src/Pages/Explore/Explore.js:22]: please gave some meaningful class name

**Review by theVivek-kumar — COMMENTED** [review]
some changes required else good to merge

File: src/Pages/Explore/Explore.js (hunk 4 above)
Comment (aiman-ansari) [reply src/Pages/Explore/Explore.js:22]: This is utility class that i am using from component library

File: src/Reducer/filterHandler.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+        all : ""
+    }
+    switch (action.type) {
+        case 'FILTER_BY_CATEGORY':
+            return{...state, category:action.payload}
+        
+        case 'ALL' :
+             return initial
+        default:
+            break;
```
Comment (rahulyadav139) [inline src/Reducer/filterHandler.js:12]: Suggestions I don't think you need to write break here.

**Review by ShrutiLakhani — APPROVED** [review]
Code looks clean. Couldn't find any issues. Good to merge 👍

**Review by rahulyadav139 — APPROVED** [review]
Appreciation everything is looking great. Clean code. Great work.

File: src/Reducer/filterHandler.js (hunk 5 above)
Comment (aiman-ansari) [reply src/Reducer/filterHandler.js:12]: thank you for point it out .I forgot to remove it

**Pull request merged**